1-org/README.md

```diff
@@ -267,29 +267,56 @@ to run the command as the Terraform service account.
 
 ### Running Terraform locally
 
-1. Change into 1-org folder.
-1. Run `cp ../build/tf-wrapper.sh .`
-1. Run `chmod 755 ./tf-wrapper.sh`
-1. Change into 1-org/envs/shared/ folder.
-1. Rename `terraform.example.tfvars` to `terraform.tfvars` and update the file with values from your environment and bootstrap.
-1. Obtain your bucket name by running the following command in the 0-bootstrap folder.
+1. Change into `1-org` folder, copy the Terraform wrapper script and ensure it can be executed.
    ```
-   terraform output gcs_bucket_tfstate
+   cd 1-org
+   cp ../build/tf-wrapper.sh .
+   chmod 755 ./tf-wrapper.sh
+   ```
+1. Change into `envs/shared` folder and rename `terraform.example.tfvars` to `terraform.tfvars`.
+   ```
+   cd envs/shared
+   mv terraform.example.tfvars terraform.tfvars
+   ```
+1. Update the file with values from your environment and 0-bootstrap output.
+1. Use `terraform output` to get the backend bucket and networks step Terraform Service Account (when using Hub and Spoke architecture) values from 0-bootstrap output.
+   ```
+   export backend_bucket=$(terraform -chdir="../../../0-bootstrap/" output gcs_bucket_tfstate | tr -d '"')
+    echo "backend_bucket = ${backend_bucket}"
+
+   networks_step_terraform_sa_email=$(terraform -chdir="../../../0-bootstrap/" output networks_step_terraform_service_account_email | tr -d '"')
+   echo "networks_step_terraform_service_account_email = ${networks_step_terraform_sa_email}"
+   ```
+1. Also update `backend.tf` with your backend bucket from 0-bootstrap output.
    ```
-1. Update `backend.tf` with your bucket from bootstrap.
+   for i in `find -name 'backend.tf'`; do sed -i "s/UPDATE_ME/${backend_bucket}/" $i; done
    ```
-   for i in `find -name 'backend.tf'`; do sed -i 's/UPDATE_ME/<YOUR-BUCKET-NAME>/' $i; done
+1. Return to  `1-org` folder
+   ```
+   cd ../../../1-org
    ```
 
 We will now deploy our environment (production) using this script.
 When using Cloud Build or Jenkins as your CI/CD tool each environment corresponding to a branch is the repository for 1-org step and only the corresponding environment is applied.
 
 To use the `validate` option of the `tf-wrapper.sh` script, please follow the [instructions](https://cloud.google.com/docs/terraform/policy-validation/validate-policies#install) to install the terraform-tools component.
 
-1. Export the projects (`terraform-org-sa`) service account for impersonation `export GOOGLE_IMPERSONATE_SERVICE_ACCOUNT="<IMPERSONATE_SERVICE_ACCOUNT>"`
-1. Run `./tf-wrapper.sh init production`.
+1. Use `terraform output` to get the Cloud Build project ID and the organization step Terraform Service Account from 0-bootstrap output. An environment variable `GOOGLE_IMPERSONATE_SERVICE_ACCOUNT` will be set using the Terraform Service Account to enable impersonation.
+   ```
+   export CLOUD_BUILD_PROJECT_ID=$(terraform -chdir="../0-bootstrap/" output cloudbuild_project_id | tr -d '"')
```

Comment:
We can remove need for translate and use -raw instead
```suggestion
   export CLOUD_BUILD_PROJECT_ID=$(terraform -chdir="../0-bootstrap/" -raw output cloudbuild_project_id)
```

Reply:
```suggestion
   export CLOUD_BUILD_PROJECT_ID=$(terraform -chdir="../0-bootstrap/" output -raw cloudbuild_project_id)
```
`-raw` option must be after `output`command

Reply:
fixed